static/app/views/issueDetails/groupFeatureFlags/useGroupSuspectFlagScores.tsx

```diff
@@ -32,12 +32,14 @@ export function useGroupSuspectFlagScores({
 }) {
   const organization = useOrganization();
   const {selection} = usePageFilters();
-  const query = {
-    environment: environment ?? selection.environments,
-    statsPeriod: statsPeriod ?? selection.datetime.period,
-    start: start ?? selection.datetime.start?.toString(),
-    end: end ?? selection.datetime.end?.toString(),
-  };
+  const query = Object.fromEntries(
+    Object.entries({
+      environment: environment ?? selection.environments,
+      statsPeriod,
+      start,
+      end,
+    }).filter(([_, value]) => !!value)
```

Comment:
```suggestion
    }).filter(Boolean)
```
might work, but double check

Reply:
Wait this doesn't work because we're looking at the key val pair, not just val

Reply:
oops okay feel free to revert